### PR TITLE
Clarification of scaling out on brokers.

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: nservicebus/scalability-and-ha/scale-out.md [hunk 1]
```diff
@@ -60,16 +60,16 @@ WARNING: When using this scaling out technique in a mixed version environment ma
 
 ### Broker transports
 
-The main difference when using broker transports is that queues are not attached to machines but are rather maintained by a central server (or cluster of servers). When running on a broker transport such as RabbitMQ or SQL Server, it is enough to specify the logical routing:
+The main difference when using broker transports is that queues are not attached to machines but rather are maintained by a central server (or cluster of servers). Such design enables usage of the Competing Consumers pattern for scaling out processing power. All instances have empty IDs and connect to the same infrastructure queue. When running on a broker transport such as RabbitMQ or SQL Server, it is enough to specify the logical routing:
```
Comment (weralabaj) [inline nservicebus/scalability-and-ha/scale-out.md]: Isn't that relevant only for V6?

Comment (weralabaj) [inline nservicebus/scalability-and-ha/scale-out.md]: Or you talk about internal infrastructure stuff?

File: nservicebus/scalability-and-ha/scale-out.md [hunk 2]
```diff
@@ -60,16 +60,16 @@ WARNING: When using this scaling out technique in a mixed version environment ma
...
 
 snippet:Routing-StaticRoutes-Endpoint-Broker
 
-Scaling out can be done, as in Version 5, by `xcopy`-ing the binaries. In this case all instances have empty IDs and there is only one queue in use.
+New instances can be deployed simply by `xcopy`-ing the binaries to another machine/folder.
```
Comment (weralabaj) [inline nservicebus/scalability-and-ha/scale-out.md]: > simply

 skip

> machine/folder

machine or folder

File: nservicebus/scalability-and-ha/scale-out.md (hunk 1 above)
Comment (SzymonPobiega) [inline nservicebus/scalability-and-ha/scale-out.md]: It is relevant for both but I decided to explicitly split this article into V5 and V6 parts. The usage of competing consumers pattern is described in the V5 part in both SQL/Rabbit and Azure sections.

Comment (weralabaj) [inline nservicebus/scalability-and-ha/scale-out.md]: My bad, didn't read the full article :D

**Pull request merged**